root/etc/e-smith/db/configuration/migrate/MigrateAmavisdPropertyToRspamd

```diff
@@ -0,0 +1,40 @@
+{
```

Comment:
An unwritten naming convention for migrate fragments is `issueXXXX.ns7`: the migrate fragment will be removed on ns8. Add more info in a comment block here.